Description: Fix resend_on_timeout: if_offline option in mod_stream_mgmt
Author: Holger Weiss <holger@zedat.fu-berlin.de>
Applied-Upstream: 17.04+1, https://github.com/processone/ejabberd/compare/120682ec...56a4bf8f
Last-Update: 2017-04-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
diff --git a/src/mod_stream_mgmt.erl b/src/mod_stream_mgmt.erl
index a1d1ba2..64cb0fd 100644
--- a/src/mod_stream_mgmt.erl
+++ b/src/mod_stream_mgmt.erl
@@ -522,7 +522,7 @@ route_unacked_stanzas(#{mgmt_state := MgmtState,
 			  Resend when is_boolean(Resend) ->
 			      Resend;
 			  if_offline ->
-			      case ejabberd_sm:get_user_resources(User, Resource) of
+			      case ejabberd_sm:get_user_resources(User, LServer) of
 				  [Resource] ->
 				      %% Same resource opened new session
 				      true;
